src/providers/bitcoin/BitcoinUtil.js

```diff
@@ -69,10 +69,41 @@ function reverseBuffer (src) {
   return buffer
 }
 
+function scriptNumSize (i) {
+  return i > 0x7fffffff ? 5
+  : i > 0x7fffff ? 4
+  : i > 0x7fff ? 3
+  : i > 0x7f ? 2
+  : i > 0x00 ? 1
+  : 0
+}
+
+function scriptNumEncode (number) {
```

Comment:
Little bit uncomfortable with writing something like this ourselves, could we have used a lib? could do with a few test cases otherwise

Reply:
This is copied from bitcoinjslib so should be safe